packages/core/functions/src/protocol/protocol.ts

```diff
@@ -146,10 +149,31 @@ class FunctionContext extends Resource {
       ? CredentialsService.layerFromDatabase().pipe(Layer.provide(dbLayer))
       : CredentialsService.configuredLayer([]);
     const functionInvocationService = MockedFunctionInvocationService;
-    const aiService = AiService.notAvailable;
     const tracing = TracingService.layerNoop;
 
-    return Layer.mergeAll(dbLayer, queuesLayer, credentials, functionInvocationService, aiService, tracing);
+    const aiLayer = this.context.services.functionsService
+      ? AiModelResolver.AiModelResolver.buildAiService.pipe(
+          Layer.provide(
+            AnthropicResolver.make().pipe(
+              Layer.provide(
+                AnthropicClient.layer({
+                  // TODO(dmaretskyi): Read endpoint from config/settings.
+                  apiUrl: 'https://ai-service.dxos.workers.dev/provider/anthropic',
```

Comment:
does it matter what endpoint it is since you're passing the request through the service binding?
I'd say its better to put it as `http://internal/....` if the hostname doesn't count anyway